src/DotLiquid.Tests/Tags/RegisterFiltersTests.cs

```diff
@@ -0,0 +1,43 @@
+using DotLiquid.Exceptions;
+using DotLiquid.Tags;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace DotLiquid.Tests.Tags
+{
+    [TestFixture]
+    public class RegisterFiltersTests
+    {
+        [Test]
+        public void TestInitialise()
+        {
+            new RegisterFilters().Initialize("register_filters", "DotLiquid.ShopifyFilters", null); // no quotes
```

Comment:
Why should no quotes be accepted as if it was quoted? This stops the use of variables in the name - which if you think there is no purpose to then we don't need quoted versions. I vote supporting quoted versions only to be consistent with expected syntaxes in tags. Also, there is no explicit test here, just testing that no error is thrown?

Reply:
Modified to render markup using the Variable class.